Tests/CucumberMessages.CompatibilityTests/CucumberMessages.Tests.csproj

```
@@ -0,0 +1,262 @@
+    <Compile Remove="Samples\Resources\retry-xRetry\retry_xRetry.cs" />
+    <Compile Remove="Samples\Resources\rules\rules.cs" />
+    <Compile Remove="Samples\Resources\skipped\skipped.cs" />
+    <Compile Remove="Samples\Resources\stack-traces\stack_traces.cs" />
+    <Compile Remove="Samples\Resources\test-run-hooks\test_run_hooks.cs" />
+    <Compile Remove="Samples\Resources\undefined\undefined.cs" />
+    <Compile Remove="Samples\Resources\unknown-parameter-type\unknown_parameter_type.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="Samples\Resources\ambiguous\ambiguous.feature" />
+    <None Remove="Samples\Resources\ambiguous\ambiguous.feature.ndjson" />
+    <None Remove="Samples\Resources\attachments\attachments.feature" />
+    <None Remove="Samples\Resources\attachments\attachments.feature.ndjson" />
+    <None Remove="Samples\Resources\background\background.feature" />
+    <None Remove="Samples\Resources\background\background.feature.ndjson" />
+    <None Remove="Samples\Resources\cdata\cdata.feature" />
+    <None Remove="Samples\Resources\cdata\cdata.feature.ndjson" />
+    <None Remove="Samples\Resources\data-tables\data_tables.feature" />
+    <None Remove="Samples\Resources\data-tables\data_tables.feature.ndjson" />
+    <None Remove="Samples\Resources\empty\empty.feature" />
+    <None Remove="Samples\Resources\empty\empty.feature.ndjson" />
+    <None Remove="Samples\Resources\examples-tables\examples_tables.feature" />
+    <None Remove="Samples\Resources\examples-tables\examples_tables.feature.ndjson" />
+    <None Remove="Samples\Resources\ExternalBindingAssemblies\ExternalBinding.feature" />
+    <None Remove="Samples\Resources\hooks\hooks.feature" />
+    <None Remove="Samples\Resources\hooks\hooks.feature.ndjson" />
+    <None Remove="Samples\Resources\minimal\minimal.feature" />
+    <None Remove="Samples\Resources\minimal\minimal.feature.ndjson" />
+    <None Remove="Samples\Resources\pending\pending.feature" />
+    <None Remove="Samples\Resources\pending\pending.feature.ndjson" />
+    <None Remove="Samples\Resources\rules\parameter-types.feature" />
+    <None Remove="Samples\Resources\rules\parameter-types.feature.ndjson" />
+    <None Remove="Samples\Resources\rules\rules.feature" />
+    <None Remove="Samples\Resources\rules\rules.feature.ndjson" />
+    <None Remove="Samples\Resources\skipped\skipped.feature" />
+    <None Remove="Samples\Resources\skipped\skipped.feature.ndjson" />
+    <None Remove="Samples\Resources\stack-traces\stack_traces.feature" />
+    <None Remove="Samples\Resources\stack-traces\stack_traces.feature.ndjson" />
+    <None Remove="Samples\Resources\undefined\undefined.feature" />
+    <None Remove="Samples\Resources\undefined\undefined.feature.ndjson" />
+    <None Remove="Samples\Resources\unknown-parameter-type\unknown_parameter_type.feature" />
+    <None Remove="Samples\Resources\unknown-parameter-type\unknown_parameter_type.feature.ndjson" />
```

Could this be simplified?

```suggestion
    <None Remove="Samples\Resources\**\*.feature" />
    <None Remove="Samples\Resources\**\*.feature.ndjson" />
```

---

I've adopted this suggestion.